### ISSUE TITLE
protected instead of private

### ISSUE THREAD

**Issue by espHorst** [adafruit/Adafruit_BME280_Library#46]
Dear all,
would it be an option to change the private methods and variables to protected?
I would like to access the private methods in an inherited class.
Best regards,
espHorst

**Comment by ladyada** [comment]
kinda odd class to need to inherit - is it something not implemented here? :)

**Comment by espHorst** [comment]
I added the BME280 to an existing Arduino Mini Pro hardware. Now I'm running out of program memory. Dropping the float library solves the issue - thus I changed the return value of the methods
```
        //returns: kelvin * 10
        uint16_t readTemperature(void);
        //returns: pascal * 10
        uint16_t readPressure(void);
        //returns: %
        uint8_t readHumidity(void);
```
and everything is fine. As I'm not sure that anybody else needs this I just wanted to inherit the existing class. This requires protected instead of private. 
If you think that more people need alternative methods returning integer values I'm happy to provide a PR adding the integer functions.
Best regards,
espHorst
